tests/eas/load_tracking.py

```diff
@@ -216,3 +229,65 @@ def test_task_load_avg(self, experiment, tasks):
         this test does the same as test_task_util_avg but for load_avg.
         """
         return self._test_signal(experiment, tasks, 'load_avg')
+
+class CpuInvarianceTest(_LoadTrackingBase):
+    """
+    Goal
+    ====
+    Basic check for CPU invariant load tracking
+
+    Detailed Description
+    ====================
+    This test runs the same workload on one CPU of each type in the system. The
+    trace is then examined to estimate an expected mean value for util_avg for
+    each CPU's workload. The util_avg value is extracted from scheduler trace
+    events and its mean is compared with the expected value (ignoring the first
+    300ms so that the signal can stabilize). The test fails if the observed mean
+    is beyond a certain error margin from the expected one. load_avg is then
+    similarly compared with the expected util_avg mean, under the assumption
+    that load_avg should equal util_avg when system load is light.
+
+    Expected Behaviour
+    ==================
+    Load tracking signals are scaled so that the workload results in roughly the
+    same util & load values regardless of compute power of the CPU used.
```

Comment:
I would also add here again something like
_"Moreover. the load signal is verified to be similar to the utilization one when running on light load conditions."_